vivarium/core/experiment.py

```diff
@@ -638,6 +632,42 @@ def apply_update(self, update, process_topology=None, state=None):
                     target.apply_defaults()
                     target.set_value(state)
 
+            move_entries = update.pop('_move', None)
```

Comment:
Also, can we add `_move` to the docstring? I think there's a spot in the documentation that covers all the special keys that should also be updated